src/classes/ERR_Handler.cls

```diff
@@ -276,7 +287,6 @@ public class ERR_Handler {
             }
 
             if (UTIL_CustomSettingsFacade.getErrorSettings().Store_Errors_On__c) {
-               UTIL_Debug.debug('**** Storing errorRecords: ' + errorRecords);
                Database.insert(errorRecords, false);
```

Comment:
could you put this UTIL_Debug.debug() back.  we always remove system.debug calls.  but UTIL_Debug calls are going thru our cover that has an NPSP Setting to turn on/off, so that customers can enable it to get more info in debug logs to help figure out problems.  This one specifically is an important one to keep enabled, since it allows us to get the actual error record into the log.